frontend/packages/dev-console/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -86,67 +98,70 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
     handleReset();
   }, [handleReset]);
 
+  const formEditor = (
+    <PipelineBuilderFormEditor
+      namespace={namespace}
+      hasExistingPipeline={!!existingPipeline}
+      taskGroup={taskGroup}
+      onTaskSelection={onTaskSelection}
+      onUpdateTasks={updateTasks}
+    />
+  );
+
+  const yamlEditor = (
+    <YAMLEditorField name="yamlData" onSave={handleSubmit} schemaModel={PipelineModel} />
```

Comment:
Need to add support for the YAML not to lose features:

The create YAML page:
![Screen Shot 2020-11-05 at 5 30 40 PM](https://user-images.githubusercontent.com/8126518/98304341-cd4a9900-1f8d-11eb-8574-2c38651eb858.png)

The YAML embedded:
![Screen Shot 2020-11-05 at 5 30 40 PM](https://user-images.githubusercontent.com/8126518/98304361-d50a3d80-1f8d-11eb-986f-a0610e52818f.png)

The user lost the ability to use Samples and Snippets.

Reply:
Moved this out of this Story and into https://issues.redhat.com/browse/ODC-5102 we'll complete this after this PR merges.